wiki/Welcome/nl.md

```diff
@@ -0,0 +1,47 @@
+# Welkom
+
+Welkom bij osu!, een free-to-win rhythm game ontworpen door peppy. Osu! bevat vier game modes: osu!standard, een simulator waarin je op cirkels klikt; osu!taiko, een drum emulator; osu!catch, een fruit vanger; en osu!mania, een synthesizer waarin je op toetsen in slaat.
+
+## Veelvoorkomende aandachtspunten
+
+- [Installatie](/wiki/Installation)
+- [Registratie](/wiki/Registration)
+- [Regels](/wiki/Rules)
+- [FAQ](/wiki/FAQ)
+- [Game Modes](/wiki/Game_Modes)
+  - ![osu!standard icon](/wiki/shared/mode/osu.png) [osu!standard](/wiki/osu!standard)
+  - ![osu!taiko icon](/wiki/shared/mode/taiko.png) [osu!taiko](/wiki/osu!taiko)
+  - ![osu!catch icon](/wiki/shared/mode/catch.png) [osu!catch](/wiki/osu!catch)
+  - ![osu!mania icon](/wiki/shared/mode/mania.png) [osu!mania](/wiki/osu!mania)
+- [Multi](/wiki/Multi)
+- [Rangschikkingscriteria](/wiki/Ranking_Criteria)
+  - ![osu!standard icon](/wiki/shared/mode/osu.png) [osu!standard](/wiki/Ranking_Criteria/osu!standard)
+  - ![osu!taiko icon](/wiki/shared/mode/taiko.png) [osu!taiko](/wiki/Ranking_Criteria/osu!taiko)
+  - ![osu!catch icon](/wiki/shared/mode/catch.png) [osu!catch](/wiki/Ranking_Criteria/osu!catch)
+  - ![osu!mania icon](/wiki/shared/mode/mania.png) [osu!mania](/wiki/Ranking_Criteria/osu!mania)
+
+## Mappen
+
+*Hoofdpagina: [Mappen](/wiki/Beatmapping)*
+
+Mappen is het proces waarin een maker een beatmap maakt. Dit proces bestaat uit: Het selecteren van muziek, de beatmap timen, de beatmap maken en het testen van de beatmap. Dit proces zou ook nog kunnen bestaan uit: het toevoegen van een video, het toevoegen van een [storyboard](/wiki/Storyboarding) en/of het toevoegen van een [zelfgemaakte skin](/wiki/Skinning). De maker van de map kan dit alleen of samen met andere gebruikers doen. *Storyboarders*, *skinners*, en/of *guest creators(gast-makers)*, zijn enkele termen die, gebaseerd op wat zij doen, verwijzen naar deze gebruikers.
+
+## Modding
+
+*Hoofdpagina: [Modding](/wiki/Modding)*
+
+Modding is het proces waar gebruikers (In het algemeen genaamd "Modders") de beatmap van iemand anders beoordelen. Modding speelt een grote rol in de kwaliteitscontrole, aangezien makers hierdoor de mogelijkheid krijgen problemen van hun beatmap op te lossen.
+
+## Skinning
+
+*Hoofdpagina: [Skinning](/wiki/Skinning)*
+
+*Zie ook: [Skinning Tutorial](/wiki/Skinning_Tutorial)*
+
+Skinning geeft iedereen de mogelijkheid om het uiterlijk van osu! te veranderen. Dit kunnen kleine veranderingen zijn, zoals het veranderen van de cursor of grote veranderingen, zoals het uiterlijk van alle gamemodes en interfaces aanpassen.
```

Comment:
Skinning geeft iedereen de mogelijkheid om het uiterlijk van osu! te veranderen. Dit kunnen kleine veranderingen (zoals het veranderen van de cursor) of grote veranderingen (zoals het uiterlijk van alle gamemodes en interfaces aanpassen) zijn.